runtime/dancebox/tests/common/mod.rs

```diff
@@ -102,7 +103,7 @@ pub fn insert_authorities_and_slot_digests(slot: u64) {
     );
 }
 
-pub fn run_block() -> RunSummary {
+pub fn run_block_with_operation(operation_fn: fn(slot: u64)) -> RunSummary {
```

Comment:
```suggestion
pub fn run_block_with_operation<F: FnOnce(u64)>(operation_fn: F) -> RunSummary {
```

This should be generic to avoid compile errors when the closure captures some state